google-cloud-clients/google-cloud-language/samples/src/main/java/com/google/cloud/examples/language/v1/LanguageClassifyGcs.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// DO NOT EDIT! This is a generated sample ("Request",  "language_classify_gcs")
+// sample-metadata:
+//   title: Classify Content (GCS)
+//   description: Classifying Content in text file stored in Cloud Storage
+//   usage: gradle run -PmainClass=com.google.cloud.examples.language.v1.LanguageClassifyGcs [--args='[--gcs_content_uri "gs://cloud-samples-data/language/classify-entertainment.txt"]']
+
+package com.google.cloud.examples.language.v1;
+
+import com.google.cloud.language.v1.ClassificationCategory;
+import com.google.cloud.language.v1.ClassifyTextRequest;
+import com.google.cloud.language.v1.ClassifyTextResponse;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+public class LanguageClassifyGcs {
+  // [START language_classify_gcs]
+  /*
+   * Please include the following imports to run this sample.
+   *
+   * import com.google.cloud.language.v1.ClassificationCategory;
+   * import com.google.cloud.language.v1.ClassifyTextRequest;
+   * import com.google.cloud.language.v1.ClassifyTextResponse;
+   * import com.google.cloud.language.v1.Document;
+   * import com.google.cloud.language.v1.LanguageServiceClient;
+   */
+
+  /**
+   * Classifying Content in text file stored in Cloud Storage
+   *
+   * @param gcsContentUri Google Cloud Storage URI where the file content is located. e.g.
+   *     gs://[Your Bucket]/[Path to File] The text file must include at least 20 words.
+   */
+  public static void sampleClassifyText(String gcsContentUri) {
+    try (LanguageServiceClient languageServiceClient = LanguageServiceClient.create()) {
+      // gcsContentUri = "gs://cloud-samples-data/language/classify-entertainment.txt";
+
+      // Available types: PLAIN_TEXT, HTML
```

Comment:
+1 to doing this. Additionally, it would be great if a link to someplace in the docs that would give more context on what these values are (Particularly for the libraries that don't have an enum type, and instead just use magic string values).